nh_eobs_mental_health/models/nh_eobs_ward_dashboard.py

```diff
@@ -31,7 +31,13 @@ class NHClinicalWardDashboard(orm.Model):
         'capacity_count': fields.integer('Free Beds'),
         'workload_count': fields.integer('All patients associated with ward'),
         'on_ward_count': fields.integer(
-            'All patients in bed and not on Patient Monitoring Exception')
+            'All patients in bed and not on Patient Monitoring Exception'),
+        'refused_obs_count': fields.integer(
+            'All patients currently refusing observations')
+    }
+
+    _defaults = {
+        'refused_obs_count': 0
     }
```

Comment:
The SQL will not always return a row if there is not results for the location. I'll investigate to see if this means that `None` is returned instead of `0` but I think this is standard Odoo behaviour